content/src/content/jcr_root/apps/core/wcm/components/tableOfContent/v1/tableOfContent/item.html

```diff
@@ -0,0 +1,25 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<template data-sly-template.item="${@ item='The table of content item'}"
+          data-sly-use.groupTemplate="group.html"
+          data-sly-use.itemContentTemplate="itemContent.html">
+    <li class="cmp-tableOfContent__item cmp-tableOfContent__item--level-${item.level}">
+        <sly data-sly-call="${itemContentTemplate.itemContent @ item = item}"></sly>
+        <sly  data-sly-call="${groupTemplate.group @ items = item.items}"></sly>
```

Comment:
We don't have nested items, please remove!

Reply:
Sure, we do need nesting: H2 should appear nested below H1.
See how the TOC is built here, this markup should serve as our guideline:
https://www.w3.org/TR/WCAG20/#contents